Utils/Game.py

```diff
@@ -1,13 +1,337 @@
+        self.mouse_down = False
+        self.letters = ['A', 'B', 'C', 'D', 'E', 'F', 'G', 'H']
+        self.fps_counter = 0
+        self.mate = False
+        self.stalemate=False
+        self.promoting_pieces = None
+
+    def load_images(self):
+        pieces = ['wR', 'wN', 'wB', 'wQ', 'wK', 'wP', 'bR', 'bN', 'bB', 'bQ', 'bK', 'bP']
+        for piece in pieces:
+            self.IMAGES[piece] = pygame.transform.scale(pygame.image.load('Utils/Images/' + piece + '.png'),
+                                                        (self.SQUARE_SIZE, self.SQUARE_SIZE))
+
+    def draw_board(self):
+        colors = [self.WHITE, self.GRAY]
+        pygame.draw.rect(self.screen, self.BLACK,
+                         (self.X_OFFSET - 1, self.Y_OFFSET - 1, self.BOARD_SIZE + 2, self.BOARD_SIZE + 2), 2)
+        for r in range(self.ROW):
+            for c in range(self.COLUMN):
+                color = colors[((r + c) % 2)]
+                pygame.draw.rect(self.screen, color,
+                                 (self.X_OFFSET + c * self.SQUARE_SIZE, self.Y_OFFSET + r * self.SQUARE_SIZE,
+                                  self.SQUARE_SIZE, self.SQUARE_SIZE))
+
+    def draw_pieces(self):
+        for r in range(self.ROW):
+            for c in range(self.COLUMN):
+                piece = self.gb.board[r][c]
+                if piece is not None:
+                    self.screen.blit(self.IMAGES[piece.get_identificator()],
+                                     pygame.Rect(self.X_OFFSET + c * self.SQUARE_SIZE,
+                                                 self.Y_OFFSET + r * self.SQUARE_SIZE,
+                                                 self.SQUARE_SIZE, self.SQUARE_SIZE))
+
+    # we are never getting here
+    def draw_possible_moves(self):
+        for r, c in self.moves:
+            move_surface = pygame.Surface((self.SQUARE_SIZE, self.SQUARE_SIZE), pygame.SRCALPHA)
+            move_surface.fill(self.GREEN)
+            self.screen.blit(move_surface, (self.X_OFFSET + c * self.SQUARE_SIZE, self.Y_OFFSET + r * self.SQUARE_SIZE))
+
+        for r, c in self.attack_moves:
+            attack_surface = pygame.Surface((self.SQUARE_SIZE, self.SQUARE_SIZE), pygame.SRCALPHA)
+            attack_surface.fill(self.RED)
+            self.screen.blit(attack_surface,
+                             (self.X_OFFSET + c * self.SQUARE_SIZE, self.Y_OFFSET + r * self.SQUARE_SIZE))
+
+    def show_checks(self):
+        r, c = None, None
+        if self.gb.check_white:
+            r, c = self.gb.white_king_position
+        elif self.gb.check_black:
+            r, c = self.gb.black_king_position
+        else:
+            return
+        move_surface = pygame.Surface((self.SQUARE_SIZE, self.SQUARE_SIZE), pygame.SRCALPHA)
+        move_surface.fill(self.DARK_RED)
+        self.screen.blit(move_surface, (self.X_OFFSET + c * self.SQUARE_SIZE, self.Y_OFFSET + r * self.SQUARE_SIZE))
+
+    def show_message(self, color, message):
```

Comment:
You should use annotations for all the methods. 

Example:

```
def show_message(self, color: Color, message: List[str]) -> None:
```

I shouldn't need to analyze whole method code to verify what are the types of parameteres and what the method returns. Propagate in entire solution